app/controllers/tournaments_controller.rb

```diff
@@ -28,9 +28,20 @@ def show
   end
 
   def statistics
+    tournament = Tournament.includes(:results).find_by(id: params[:id])
+    if tournament_not_found(tournament)
```

Comment:
The name `tournament_not_found` is confusing - because it returns true if tournament is found, so it has the opposite meaning of its name

Reply:
I think you should rename it to something like `render_tournament_not_found_error`

Reply:
And also return true if tournament was NOT found
Then you can do:
`return if render_tournament_not_found_error(tournament)`

Reply:
i'd actually say that a better name would be `tournament_found?`
will return `true` if found, `false` otherwise